instructions/docs/add-the-encryption-sdk.md

```diff
@@ -130,7 +129,7 @@ Now, before storing data in the Document Bucket, it uses the AWS Encryption SDK
 
 ### Step 3: Add Decryption to `retrieve`
 
-Now that you are encrypting data before storing it, you need to decrypt it before returning it to your caller. At least for it to be useful, anyway.
+Now that the application is encrypting data before storing it, it needs to decrypt it before returning it to the caller. At least for it to be useful, anyway.
```

Comment:
still using present tense. Consider "Now that the application will encrypt data before storing it…"

Reply:
Thanks!